scenarios/Makefile

```
@@ -102,12 +102,26 @@ mn-cli: ./tmp/.env.docker
 mn-log:
 	docker logs -f mininet
 
-onos-cli:
+onos-cli: onos-cli1
+
+onos-cli1:
 	ssh -o "UserKnownHostsFile=/dev/null" -o "StrictHostKeyChecking=no" -o LogLevel=ERROR -p 8101 localhost
 
-onos-log: ./tmp/.env.docker
+onos-cli2:
+	ssh -o "UserKnownHostsFile=/dev/null" -o "StrictHostKeyChecking=no" -o LogLevel=ERROR -p 8102 localhost
+
+onos-cli3:
+	ssh -o "UserKnownHostsFile=/dev/null" -o "StrictHostKeyChecking=no" -o LogLevel=ERROR -p 8103 localhost
+
+onos-log-1: ./tmp/.env.docker
```

For consistency with the cli targets. Applies to other log targets as well.
```suggestion
onos-log1: ./tmp/.env.docker
```